kustomize/data-plane/prerequisites/knative/resources/knative-eventing.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec: {}
```

Comment:
@winklerm Is there a way to constrain which version of Knative evening is installed using this method?